Localization/fr-FR/dig_deeper.po

```diff
@@ -0,0 +1,241 @@
+msgid ""
+msgstr ""
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/dig-deeper/\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
```

Comment:
Can you please check that the source for the translation is the latest version. The latest source version for this file was updated on 2020-09-23, and is version 1.2.

Reply:
If you are wondering what has changed since 1.0: 
https://github.com/FoldingCommunity/Translate/pull/136/commits/1d81098bea42806216b444c3e2b78015ee3aa4e3
https://github.com/FoldingCommunity/Translate/pull/135/commits/16f4abcf964d403dd4efa1f2990aa4b9753cacf3

Reply:
If you are wondering waht ahs changed since 1.0:
https://github.com/FoldingCommunity/Translate/pull/136/commits/1d81098bea42806216b444c3e2b78015ee3aa4e3
https://github.com/FoldingCommunity/Translate/pull/135/commits/16f4abcf964d403dd4efa1f2990aa4b9753cacf3

Reply:
Othervise really good job. This fille is massive and is probably the biggest file that will ever exisist. Thanks for the help.

Reply:
Thank you for the heads-up and the help. Actually, the changes are already implemented in the file. We just failed to adapt the source version for the PR, as the translation was ongoing during the changes. It will be done shortly.

Reply:
I have updated the file with the changes requested